### ISSUE TITLE
Building Tier Rebalance

### ISSUE THREAD

**Issue by teagan75** [StrangeLoopGames/EcoSuggestions#1381]
Right now I feel the tiers are broken and need more love. 

Carpentry
Tier 1 - Wattle and Daub (use for dirt and wood pulp)
Tier 2 - Hewn Logs
Tier 3 - Lumber
Tier 4 - Composites

Masonry
Tier 1 - Cobblestone/Chiseled Stone (no mortar)
Tier 2 - Bricks
Tier 3 - Reinforced Concrete
Tier 4 - Ashlar

Remove the ability for the Engineers to make their own building blocks since they are so needed for machines and endgame and leave the building materials to Carpenter and Mason.

**Comment by JestinJae** [comment]
What blocks do engineers make? I only see road tiles, which aren't related to normal building.

**Comment by teagan75** [comment]
I am talking about the whole Engineering profession with all the subsets. This probably isn’t as big an issue on large servers but it is on smaller servers

**Comment by JestinJae** [comment]
Right, I was talking about the entire profession also, guess I just don't
understand.

**Comment by teagan75** [comment]
I could be wrong too. It could be advanced Smeltery for all I know. Lol
